stl/src/vector_algorithms.cpp

```diff
@@ -176,6 +177,7 @@ __declspec(noalias) void __cdecl __std_reverse_trivially_swappable_1(void* _Firs
             _Advance_bytes(_First, 16);
         } while (_First != _Stop_at);
     }
+#endif // !defined(_M_ARM64EC)

```

Comment:
If `_M_ARM64EC` contains only fallback, it should not be called at all; instead `_USE_STD_VECTOR_ALGORITHMS` should be set to 0, like for `_M_HYBRID`.

Maybe they should even not exist, like for _M_ARM.

Reply:
I think we should mimic the behavior of `_M_HYBRID` I just put this PR out as it was to get the idea out there.

Reply:
I don't agree with this; I believe that arm64EC should be treated like ARM64 for purposes of vector instructions, since it is "just" ARM64 code with an emulator-friendly ABI.